auto_guild.py

```diff
@@ -195,10 +206,12 @@ def transfer_ownership(session, user_id, guild_id):
 
         with open("guild_layout.yaml", "w") as file:
             dump(finished_guild, file)
+        print(f"Guild layout saved to guild_layout.yaml")
 
         invite_url = get_invite(session, invite_channel_id)
         print(invite_url)
         webbrowser.open(invite_url, new=2)
```

Comment:
```suggestion        
```
nitpick: I don't like when apps automatically open my browser for me

Reply:
Perhaps have it take an input asking if they want it to do so.